### PR TITLE
ui: remove extra table tag in service discovery

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

**Comment by brian-brazil** [comment]
:+1: 

I'm surprised that didn't cause problems.

**Comment by roidelapluie** [comment]
Here is the "problem" (space between federate_1 and federate_2) :
prob

**Comment by roidelapluie** [comment]
(present in last stable too:  http://demo.robustperception.io:9090/service-discovery )

File: web/ui/templates/service-discovery.html [hunk 1]
```diff
@@ -18,7 +18,6 @@
 
     <h1>Service Discovery</h1>
     <div class="table-container">
```
Comment (tijmenb) [inline web/ui/templates/service-discovery.html]: Do we still need to add this div/class? `table-container` seems only used to [add functionality and styles to tables](https://github.com/prometheus/prometheus/search?utf8=%E2%9C%93&q=table-container&type=).

Comment (roidelapluie) [reply web/ui/templates/service-discovery.html]: Indeed!

**Comment by codesome** [comment]
Thanks!

**Pull request merged**